packages/type-params/src/decode/value/index.ts

```diff
@@ -27,7 +29,7 @@ export default function decodeValue (decode: Decoder, type: Param$Type, input: U
     switch (type) {
       // TODO Pass back the actual address, not publicKey?
       case 'AccountId':
-        return u8a(input, 256, version === 'poc-1' ? 0 : 1);
+        return account(input.slice(0, 32), 256);
```

Comment:
Changes as described above

1. pass in version
2. no need to pass in bitLength